File: paradicms_etl/namespaces/exif.py
from rdflib import URIRef
from rdflib.namespace import DefinedNamespace, Namespace


class EXIF(DefinedNamespace):
    _fail = True
    _NS = Namespace("http://www.w3.org/2003/12/exif/ns#")

    height: URIRef
    width: URIRef

File: lib/models/src/vocabularies/EXIF.ts
import Namespace from "../Namespace";

export class EXIF {
  static readonly NS = Namespace("http://www.w3.org/2003/12/exif/ns#");

  // Properties
  static readonly height = EXIF.NS("height");
  static readonly width = EXIF.NS("width");
}
